alerts/notifications.py

```diff
@@ -0,0 +1,36 @@
+from django.core.mail import send_mail
+from users.models import User
+from series.models import APIShow
+
+
+class NotificationBackend:
+    """Backend for generic notifications."""
+
+    def notify(self, user: User, tv_show: APIShow):
```

Comment:
The convention has been to use `show` to refer to show (or APIShow when unambiguous) objects. Can we follow it here too?